.svnignore

```diff
@@ -42,3 +42,4 @@ docker
 bin/pre-commit-hook.js
 bin/travis_install.sh
 yarn-error.log
+extensions/**/*.{js,jsx,json,gif,png,css,jpg,jpeg,sass,scss,svg}
```

Comment:
Should we add `md` files to that list? We probably don't need to ship the different readme in there.

Reply:
I've tried to include everything here that is a frontend oriented asset, things that are clearly used to build client assets.

`.md` is a bit different, purely informative. I'm not sure whether that sort of thing should be excluded or not. If you think so I'm happy to add it to the list 🙂

Reply:
I would add .md files to that list; those files won't be very useful without the rest of the block files anyway.

Reply:
6a19bed ignores `.md` files in extensions